src/node-red/node-red.json

```diff
@@ -1,5 +1,6 @@
 {
-    "layout": "layouts/learning-resources.njk",
+    "layout": "layouts/handbook.njk",
```

Comment:
Feels dirty if the filename is `node-red.json`?

Reply:
Fixed! https://github.com/FlowFuse/website/pull/2635